.gitignore

```diff
@@ -180,3 +180,7 @@ custom-elements.json
 # Yalc
 .yalc
 yalc.lock
+
+# Compsizer
+**/compsizer-failure-report.json
+**/component-bundle-sizes.json
```

Comment:
Is this to do with Compsizer rather than bundlewatch? (so may be able to remove)